Exercise4/javascript/test3.js

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
This works but can you use recursion to solve the same problem?

Try to use recursion to solve it too